exchange/messaging.go

```diff
@@ -484,17 +486,20 @@ func DeleteKeys(keyPath string) error {
 	privFilepath := path.Join(snap_common, keyPath, privFileName)
 	pubFilepath := path.Join(snap_common, keyPath, pubFileName)
 
-	glog.V(5).Infof("Removing private key path %v, and public key path %v", privFilepath, pubFilepath)
+	cleanedPrivFilepath := filepath.Clean(privFilepath)
```

Comment:
Should apply the clean function to "func GetKeys(keyPath string)" as well to make it consistent